Dockerfile

```diff
@@ -17,11 +17,11 @@ EXPOSE 8080 8081
 CMD exec java $JAVA_OPTS \
      -Ddw.server.applicationConnectors[0].port=8080 \
      -Ddw.server.adminConnectors[0].port=8081 \
-     -Ddw.methodeArticleMapper.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.methodeContentPlaceholderMapper.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.documentStoreApi.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.concordanceApi.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.consumer.messageConsumer.queueProxyHost=http://$VULCAN_HOST \
-     -Ddw.producer.messageProducer.proxyHostAndPort=$VULCAN_HOST \
+     -Ddw.methodeArticleMapper.endpointConfiguration.primaryNodes=$METHODE_ARTICLE_MAPPER_URL \
+     -Ddw.methodeContentPlaceholderMapper.endpointConfiguration.primaryNodes=$METHODE_CPH_MAPPER_URL \
+     -Ddw.documentStoreApi.endpointConfiguration.primaryNodes=$DOC_STORE_API_URL \
+     -Ddw.concordanceApi.endpointConfiguration.primaryNodes=$CONCORDANCE_API_URL \
+     -Ddw.consumer.messageConsumer.queueProxyHost=http://$KAFKA_PROXY_URL \
```

Comment:
Producer and consumer have different names for the same kind of parameter, also one has http in front of the value and the other doesn't. Please arrange

Reply:
I'll adjust them. I'll check if the consumer/producer is a inconsistency, I see it in every mapper repo though.

Reply:
So the names are defined in a 3rd party library, so only by doing some wrapping class would work, but I don't think it's worth it.

Reply:
That's ok, leave it.

Reply:
Thanks #http